conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/DynamicInterpreter.scala

```diff
@@ -33,7 +33,7 @@ import za.co.absa.enceladus.utils.general.Algorithms
 import za.co.absa.enceladus.utils.schema.SchemaUtils
 
 object DynamicInterpreter {
-  private val log = LogManager.getLogger("enceladus.conformance.DynamicInterpreter")
+  private val log = LogManager.getLogger(this.getClass)
```

Comment:
Can we use `slf4j` interfaces rather than the underlying `log4j` implementation directly?